independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4553 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HubSpot CRM",
+    "description": "HubSpot CRM is a system for managing customer relationships and storing data. The CRM custom connector provides access to CRM objects or groups of objects, which are represented as a map of property names to values.",
+    "version": "1.0",
+    "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+    }
+  },
+  "host": "api.hubapi.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/crm/v3/objects/companies": {

```

Comment:
/crm/v3/objects/ should be base path.

Reply:
By leaving this as base path "/", additional action based on v1 and beta actions can be developed. No change